src/i18n/index.ts

```diff
@@ -0,0 +1,47 @@
+import i18n from 'i18next'
+import { initReactI18next } from 'react-i18next'
+import LanguageDetector from 'i18next-browser-languagedetector'
+
+import translation_en from './translation_en.json'
+import translation_fr from './translation_fr.json'
+
+i18n
+  .use(LanguageDetector)
+  .use(initReactI18next)
+  .init({
+    debug: true,
+    detection: {
+      order: ['querystring', 'localStorage', 'navigator', 'htmlTag'],
```

Comment:
Ideally, language detection behavior should be documented in a storybook. It would be great to have it.

Reply:
I think we could implement it later, but I'm OK with you it would be great to have it so I will take a look for this PR.

Reply:
Storybook implemented 👌